es/08_0_Expandiendo_Bitcoin_Transacciones_Otros.md

```diff
@@ -0,0 +1,21 @@
+# Capítulo ocho: Expandiendo las transacciones de Bitcoin de otras formas
```

Comment:
- `Expandiendo las transacciones de Bitcoin en otros sentidos`
- Requiere modificar LEEME. #377

Reply:
Please Could you precise what changes are required?,  and Where?

Reply:
Oh, this is OK. I'm just linking this pr with the LEEME pr to keep account of needed modifications for filenames.